src/System.Runtime.Serialization.Formatters/src/Resources/Strings.resx

```diff
@@ -129,4 +129,223 @@
   <data name="InvalidOperation_EnumOpCantHappen" xml:space="preserve">
     <value>Enumeration has either not started or has already finished.</value>
   </data>
+  <data name="Serialization_NonSerType" xml:space="preserve">
+    <value>Type '{0}' in Assembly '{1}' is not marked as serializable.</value>
+  </data>
+  <data name="Argument_DataLengthDifferent" xml:space="preserve">
+    <value>Parameters 'members' and 'data' must have the same length.</value>
+  </data>
+  <data name="ArgumentNull_NullMember" xml:space="preserve">
+    <value>Parameters 'members' and 'data' must have the same length.</value>
+  </data>
+  <data name="Serialization_UnknownMemberInfo" xml:space="preserve">
+    <value>Only FieldInfo, PropertyInfo, and SerializationMemberInfo are recognized.</value>
+  </data>
+  <data name="Serialization_NoID" xml:space="preserve">
+    <value>Object has never been assigned an objectID.</value>
+  </data>
+  <data name="ArgumentNull_Obj" xml:space="preserve">
+    <value>Object cannot be null.</value>
```

Comment:
Nit: This resource string should probably be deleted. It doesn't really add enough value to justify itself (IMO), based on the few places it's used as a custom message to `ArgumentNullException`.